src/common/TALENTS/shaman.ts

```diff
@@ -349,7 +349,7 @@ const talents = {
     maxRanks: 1,
     entryIds: [102004],
     definitionIds: [{ id: 106968, specId: 262 }],
-    manaCost: 5000,
+    manaCost: 12500,
```

Comment:
this is going to be overwritten by the next generation, but maybe that is fine if the next generation just hasn't happened yet?

Reply:
I just noticed the mana cost warning in console, not a big deal if it gets overridden.

Reply:
kk